backend/api/database/models.py

```diff
@@ -20,11 +19,9 @@ class Base(MappedAsDataclass, DeclarativeBase):
     # PostgreSQL. This is only needed for the case where a specific PostgreSQL
     # type has to be used or when we want to ensure a specific setting (like the
     # timezone below)
-    type_annotation_map = {
-        dict[str, Any]: MutableDict.as_mutable(
-            JSONB
-        ),  # transform Python Dict[str, Any] into PostgreSQL JSONB
-        list[dict[str, Any]]: MutableList.as_mutable(JSONB),
+    type_annotation_map: ClassVar = {
```

Comment:
Can you explain why you'd need this while zimfarm doesn't?

Reply:
If you mean the `ClassVar` we don't really need it, I will delete it then. 
If you mean the `type_annotation_map`, I copy it from the zimfarm:
https://github.com/openzim/zimfarm/blob/4e9ea36542d90e8835e7e4323917e6ce4f4f2b3f/dispatcher/backend/src/db/models.py#L39-L51